recipes/pylbm/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: pylbm
+  version: {{ version }}
+
+source:
+  url: https://github.com/pylbm/pylbm/archive/{{ version }}.tar.gz
+  sha256: 9a9263a5f14fbbd6a65b73c7e904c1add789f3334228f9535309d533a104b552
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+  run:
+    - python >=3.4
+    - numpy
+    - matplotlib
+    - sympy >=1.1.1,<1.2
+    - cython
+    - h5py
+    - mpi4py
+    - future
+    - colorlog
+
+test:
+  imports:
+    - pylbm
+
+about:
+  home: http://pylbm.readthedocs.io
+  license: BSD 3-Clause
```

The SPDX identifier has two dashes:
```suggestion
  license: BSD-3-Clause
```